handlers/handlers.go

```diff
@@ -1,11 +1,45 @@
 package handlers
 
 import (
-	"fmt"
-	"github.com/gorilla/mux"
+	"encoding/json"
+	"log"
 	"net/http"
+
+	"github.com/NextGenConf/SessionService/models"
+	"github.com/gorilla/mux"
 )
 
-func GetSession(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Hi there, I like %s!", mux.Vars(r)["key"])
+type Environment struct {
+	db models.SessionDatabaseHandler
+}
+
+func InitializeEnvironment() *Environment {
+	return &Environment{
+		db: models.InitializeDatabaseHandler(),
+	}
+}
+
+func (e *Environment) GetSession(w http.ResponseWriter, r *http.Request) {
```

Comment:
Not familiar with golang style, so more a question: Is it standard to use one letter variables?

Reply:
It's quite common for http handlers (not sure about the reason).

Reply:
From official google doc:
![image](https://user-images.githubusercontent.com/3309748/69239010-8e171280-0b99-11ea-9125-7443de2bf2d3.png)